### DISCUSSION TITLE
Matrix Notification Support

### DISCUSSION THREAD

**Discussion by ZicPL** [Ylianst/MeshCentral#6611] [Ideas]
Hi 👋

Yesterday I found this app for the first time. I regret that I didn't know about MeshCentral earlier. I want to say that Owner/Dev team did a great job ! I've never used npm, but I'll start learning for MeshCentral (I want to test this app). I looked to see if there was a request to add "notifications to the Matrix channel" (for discord exist), but only phrase appeared on a few issues - nothing else..I think that Open-Source apps working together is a great gesture of people who see the benefits of such cooperation. So I've a question:

### Can MeshCentral support notifications from Matrix?

This is just my little suggestion (no pressure). A positive stance from Owner/Dev Team in this thread would be great news for everyone. I'd appreciate Your response and Your time.

kind regards,

**Comment by si458** [comment]
We currently don't support matrix as a messaging method to get alerts and login codes

If you want this adding then please make create a github issue and use the enhancement request and fill it all in explain (who what when where and why 🎶) the feature

Also if possible, add any docs to developer guides or nodejs modules we could possibly use to help us

**Reply by ZicPL** [reply]
@si458 Thank You for Your quick response. I'll try to do as You said (I've never had to deal with Nodejs - only LAMP, but I'm learning more and more about the advantages of this JS technology in Backednd). Please be understanding if I make a mistake when creating an issue. I'll paste here link to created issue.

**Reply by si458** [reply]
No thats perfectly fine! Just open the issue as an enhancement request and just explain u want this support and for it to do this, and IF u find any developer docs or npmjs.com modules then include those as it just helps us out that's all

**Reply by ZicPL** [reply]
@si458 I created an [issue](https://github.com/Ylianst/MeshCentral/issues/6613) as You suggested. Let me know what You think :)

**Reply by si458** [reply]
thank you will have a look next week when i get chance!
